novawallet/Modules/DApp/DAppBrowser/DAppBrowserTabs/DAppBrowserTabsViewController.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+typealias DappBrowserTabCell = PlainBaseTableViewCell<UILabel>
```

Comment:
I saw that on the PR screenshots there is an image per cell but can't find it. Probably forget to push

Reply:
The second screenshot shows dotappstore, which appears to be tabs in a screenshot format, but it isn't. I haven't yet implemented screenshots or mature persistence using CoreData, as I want to avoid dealing with migrations later in this branch. The tab model should be properly planned first.